code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,433 @@
+		if(mob_occupant && mob_occupant.stat != DEAD)
+			to_chat(occupant, "<span class='boldnotice'>You feel cool air surround you. You go numb as your senses turn inward.</span>")
+		if(mob_occupant.client)//if they're logged in
+			despawn_world_time = world.time + (time_till_despawn * 0.1)
+		else
+			despawn_world_time = world.time + time_till_despawn
+	icon_state = "cryopod"
+
+/obj/machinery/cryopod/open_machine()
+	..()
+	icon_state = "cryopod-open"
+	density = TRUE
+	name = initial(name)
+
+/obj/machinery/cryopod/container_resist(mob/living/user)
+	visible_message("<span class='notice'>[occupant] emerges from [src]!</span>",
+		"<span class='notice'>You climb out of [src]!</span>")
+	open_machine()
+
+/obj/machinery/cryopod/relaymove(mob/user)
+	container_resist(user)
+
+/obj/machinery/cryopod/process()
+	if(!occupant)
+		return
+
+	var/mob/living/mob_occupant = occupant
+	if(mob_occupant)
+		// Eject dead people
+		if(mob_occupant.stat == DEAD)
+			open_machine()
+
+		if(!(world.time > despawn_world_time + 100))//+ 10 seconds
+			return
+
+		if(!mob_occupant.client && mob_occupant.stat < 2) //Occupant is living and has no client.
+			if(!control_computer)
+				find_control_computer(urgent = TRUE)//better hope you found it this time
+
+			despawn_occupant()
+
+// This function can not be undone; do not call this unless you are sure
+/obj/machinery/cryopod/proc/despawn_occupant()
+	var/mob/living/mob_occupant = occupant
+
+	if(istype(SSticker.mode, /datum/game_mode/cult))//thank
+		if("sacrifice" in SSticker.mode.cult_objectives)
+			var/list/possible_targets = list()
+			for(var/mob/living/carbon/human/H in GLOB.player_list)
+				if(H.mind && !is_convertable_to_cult(H) && !iscultist(H))
+					possible_targets += H
+
+			possible_targets -= mob_occupant.mind
+			if(!possible_targets.len)
+				message_admins("Cult Sacrifice: Could not find unconvertable target, checking for convertable target.")
+				for(var/mob/living/carbon/human/player in GLOB.player_list)
+					if(player.mind && !iscultist(player))
+						possible_targets += player.mind
+
+			if(possible_targets.len > 0)
+				GLOB.sac_mind = pick(possible_targets)
+				if(!GLOB.sac_mind)
+					message_admins("Cult Sacrifice: ERROR -  Null target chosen!")
+				else
+					var/datum/job/sacjob = SSjob.GetJob(GLOB.sac_mind.assigned_role)
+					var/datum/preferences/sacface = GLOB.sac_mind.current.client.prefs
+					var/icon/reshape = get_flat_human_icon(null, sacjob, sacface)
+					reshape.Shift(SOUTH, 4)
+					reshape.Shift(EAST, 1)
+					reshape.Crop(7,4,26,31)
+					reshape.Crop(-5,-3,26,30)
+					GLOB.sac_image = reshape
+
+					for(var/datum/mind/H in SSticker.mode.cult)
+						if(H.current)
+							to_chat(H.current, "<span class='danger'>Nar'Sie</span> murmurs, <span class='cultlarge'>[occupant] is beyond your reach. Sacrifice [GLOB.sac_mind.current] instead...</span></span>")
+
+	//Update any existing objectives involving this mob.
+	for(var/datum/objective/O in GLOB.objectives)
```

Comment:
Silly edgecase - but if there are no other possible targets... it just keeps your original "assassinate Jeff" objective.